base-local/database/scripts/01_create_db_and_users.sql

```diff
@@ -1,10 +1,11 @@
 CREATE DATABASE lcpserver;
 CREATE user lcpserver identified by 'secretpassword';
-GRANT ALL ON lcpserver.* TO 'lcpserver'@'%';
+GRANT ALL ON lcpserver.* TO 'lcpserver' @'%';
```

Comment:
Is the space here (and below) just an auto-format change?

Reply:
It might be an auto-format change but I fixed it